app/Http/Controllers/Api/DatabasesController.php

```
@@ -2749,4 +2808,262 @@ public function action_restart(Request $request)
+            return response()->json(['message' => 'Database not found.'], 404);
+        }
+
+        $this->authorize('view', $database);
+
+        $restores = DatabaseRestore::where('database_id', $database->id)
+            ->where('database_type', $database->getMorphClass())
+            ->orderBy('created_at', 'desc')
+            ->get();
+
+        return response()->json($restores);
+    }
```

PHPMD (2.15.0)

The method list_restores is not named in camelCase. (undefined)

(CamelCaseMethodName)